main/store/migrations/index.js

```diff
@@ -856,6 +848,54 @@ const migrations = {
       }
     }
 
+    return initial
+  },
+  34: (initial) => {
+    // Add any missing nativeCurrency name values
+    // Base Görli (84531) value added in #33
+    const nativeCurrencyMap = {
+      1: {
+        name: 'Ether',
+        symbol: 'ETH'
+      },
+      5: {
+        name: 'Görli Ether',
+        symbol: 'görETH'
+      },
+      10: {
+        name: 'Ether',
+        symbol: 'ETH'
+      },
+      100: {
+        name: 'xDAI',
+        symbol: 'xDAI'
+      },
+      137: {
+        name: 'Matic',
+        symbol: 'MATIC'
+      },
+      42161: {
+        name: 'Ether',
+        symbol: 'ETH'
+      },
+      11155111: {
+        name: 'Sepolia Ether',
+        symbol: 'sepETH'
+      }
+    }
+
+    Object.values(initial.main.networks.ethereum).forEach((network) => {
+      const { id } = network
+      const { name = '', symbol = '' } = nativeCurrencyMap[id] || {}
+      const nativeCurrency = initial.main.networksMeta.ethereum[id].nativeCurrency
+
+      initial.main.networksMeta.ethereum[id].nativeCurrency = {
+        ...nativeCurrency,
+        name: nativeCurrency.name || name,
```

Comment:
This is true

Reply:
I think this would assign the name to what was in `nativeCurrency` even if it were falsy, for example overwriting the new value with empty string, for example